keystrokes/src/components/Character/CharacterCreationModal.js

```diff
@@ -0,0 +1,118 @@
+import React, {useState} from 'react';
+import {Modal, Button, TextField, FormControl, InputLabel, Select, MenuItem, Grid} from '@material-ui/core';
+import { characterDetails } from '../../config/constants';
+import styleGuide from '../../stylesheets/style';
+
+function CharacterCreationModal({handleSubmit, isOpen, setIsOpen}) {
+    const [characterName, setCharacterName] = useState("");
+    const [characterClass, setCharacterClass] = useState("");
+    const [characterRace, setCharacterRace] = useState("");
+    const [characterSpec, setCharacterSpec] = useState("");
+	const classes = styleGuide();
+
+    const handleChangeName = (event) => {
```

Comment:
the next four functions look almost identical to me which makes me wonder if they are all necessary. in this case you could inline these arrow functions in the onChange prop and just call the setCharacter___ 

also try and prefer named functions 
function handleChangeName in this case